codegen_glibmm/templates/common.cpp.templ

```diff
@@ -20,7 +22,8 @@ static void throw_func(GError *gerror)
     const gchar *message;
 
     if (g_str_has_prefix(gerror->message, prefix)) {
-        message = strstr(gerror->message + sizeof(prefix), ": ") + 2;
+        const char *colon = std::strstr(gerror->message + sizeof(prefix), ": ");
```

Comment:
so at this point we are sure that `gerror->message` has ":" in it since the prefix is `GDBus.Error:`
I assume this is a safety in case a developers mistakenly change prefix.

Reply:
This is a different colon: the error message is of the form

    GDBus.Error:com.example.MyService.InternalError: something bad

so we want only the "something bad" part.